docs/updating-the-kit.md

```diff
@@ -1,17 +1,96 @@
 # Updating the kit
 
+There are two ways you could upgrade the prototype kit. If you're confident in your use of the command line interface such as terminal then read the 'Upgrading via the command line' instructions. If you'd prefer not to or are not using the command line to manage your prototype then read the 'Upgrading manually' instructions. 
```

Comment:
Elsewhere, we assume our primary user is someone who does not have technical experience - so I think we should do that here too and mention the manual instructions first as the common case.